js/index.js

```diff
@@ -0,0 +1,50 @@
+const buttonAdd = document.querySelector('#add-btn');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const bookDisplay = document.querySelector('#book-section');
+
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+const addBook = (title, author) => {
+  books.push({ title, author });
+  localStorage.setItem('books', JSON.stringify(books));
+  // eslint-disable-next-line no-use-before-define
+  render();
+};
+
+const removeBook = (index) => {
+  books.splice(index, 1);
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make use array methods that doesn't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) and so on